components/molecule/select/src/index.scss

```diff
@@ -5,9 +5,13 @@
 
 $base-class: '.sui-MoleculeSelect';
 $class-input: '#{$base-class}-inputSelect';
+$class-select-atom-input: '.sui-AtomInput';
+$class-select-atom-input-input: '#{$class-select-atom-input}-input';
+$class-select-atom-input-tags: '#{$class-select-atom-input}--withTags';
 
 $z-select-list: $z-navigation !default;
-$mr-select-list-arrow: -34px !default;
+$mr-select-list-arrow: -32px !default;
+$pr-select-atom-input-tags: 32px !default;
```

Comment:
First, none of these values is good for this case? https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_spacing.scss

Reply:
Fixed!